app/controllers/api/v1/merchants/find_all_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::Merchants::FindAllController < ApplicationController
   def index
     if params["name"]
-      render json: Merchant.where(name: params["name"].capitalize)
+      render json: Merchant.where("lower(name) = ?", params["name"].downcase)
```

Comment:
This is going to be super helpful for where I'm at, thank you for taking this on!